core/schemas/graph.py

```diff
@@ -1,17 +1,29 @@
 import datetime
 from typing import ClassVar
 
-from pydantic import BaseModel
-
 from core import database_arango
+from pydantic import BaseModel, computed_field
 
 
 # Database model
+# Relationship and TagRelationship do not inherit from YetiModel
+# because they represent and id in the form of collection_name/id
 class Relationship(BaseModel, database_arango.ArangoYetiConnector):
     _collection_name: ClassVar[str] = "links"
     _type_filter: ClassVar[str | None] = None
 
-    id: str | None = None
+    __id: str | None = None
+
+    def __init__(self, **data):
+        super().__init__(**data)
+        self.__id = data.get("__id", None)
+
+    @computed_field(return_type=str)
+    @property
+    def id(self):
+        return self.__id
+
+#    id: str | None = None
```

Comment:
```suggestion
```

Reply:
Fixed in c7a5a4dda47ad97bd9023d430c0f59511d837475